ui/src/app/projects/projects.component.ts

```diff
@@ -95,21 +95,20 @@ export class ProjectsComponent implements OnInit {
   }
 
   navigateJobs() {
+    const extras = {
```

Comment:
nit: name it something like projectQueryParams

Reply:
Thanks! This value is of `[NavigationExtra](https://angular.io/api/router/NavigationExtras)` class defined by Angular. Only the `queryParams` is about query. Plus, it is originally called `extra`, which might make more sense here.